### ISSUE TITLE
Datepicker Table not Accessible via VoiceOver (Safari)

### ISSUE THREAD

**Issue by ntsadu** [airbnb/react-dates#1419]
Browser: Safari v11.1
Library: react-dates v18.1

When testing the datepicker, VoiceOver doesn't navigate the table using VO + up/down. Only VO + left/right works.. To be considered a **fully-accessible** datepicker I think both should work.

Please reference: https://www.w3.org/TR/wai-aria-1.1/#grid

I have the code changes to make this work as a grid for VoiceOver users, however the role="button" in the <td> elements is sacrificed for role="gridcell". Let me know if you approve and I'll submit a PR

**Comment by backwardok** [comment]
That seems like a good change! Thanks for the suggestion. Assigning it back to you since you said you have code changes ready for it.
